platform-ui-skin/src/main/webapp/skin/less/helpers.less

```diff
@@ -64,7 +64,7 @@
   color: @textLightColor !important;
 }
 .light-grey-background {
-  background-color: @greyColorOpacity1 !important;
+  background-color: @lightGrey !important;
```

Comment:
This change will impact other components, please define another class if you need this color to be used.

Reply:
done